es/cheatsheet-deep-learning.md

```diff
@@ -1,291 +1,291 @@
 
 <br>
 
 **31. Definitions**
 
-&#10230;
+&#10230; Definiciones
 
 <br>
 
 **32. Markov decision processes ― A Markov decision process (MDP) is a 5-tuple (S,A,{Psa},γ,R) where:**
 
-&#10230;
+&#10230; Procesos de decisión de Markov - Un proceso de decisión de Markov (MDP por sus siglas en inglés) es una 5-tupla (S,A,{Psa},γ,R) donde:
 
 <br>
 
 **33. S is the set of states**
 
-&#10230;
+&#10230; S es el conjunto de estados
 
 <br>
 
 **34. A is the set of actions**
 
-&#10230;
+&#10230; A es el conjunto de acciones
 
 <br>
 
 **35. {Psa} are the state transition probabilities for s∈S and a∈A**
 
-&#10230;
+&#10230; {Psa] son las probabilidades de transición de estado para s∈S y a∈A
 
 <br>
 
 **36. γ∈[0,1[ is the discount factor**
 
-&#10230;
+&#10230; γ∈[0,1[ es el factor de descuento
 
 <br>
 
 **37. R:S×A⟶R or R:S⟶R is the reward function that the algorithm wants to maximize**
 
-&#10230;
+&#10230; R:S×A⟶R o R:S⟶R es la función recompensa que el algoritmo pretende maximizar**
 
 <br>
 
 **38. Policy ― A policy π is a function π:S⟶A that maps states to actions.**
 
-&#10230;
+&#10230; Política - Una política π es una función π:S⟶A que asigna estados a acciones.
 
 <br>
 
 **39. Remark: we say that we execute a given policy π if given a state a we take the action a=π(s).**
 
-&#10230;
+&#10230; Observación: decimos que ejecutamos una política π dada si dado un estado a tomamos la acción a=π(s).
 
 <br>
 
 **40. Value function ― For a given policy π and a given state s, we define the value function Vπ as follows:**
 
-&#10230;
+&#10230; Función valor - Para una política dada π y un estado dado s, definimos el valor de la función Vπ de la siguiente manera:
 
 <br>
 
 **41. Bellman equation ― The optimal Bellman equations characterizes the value function Vπ∗ of the optimal policy π∗:**
 
-&#10230;
+&#10230; La ecuación de Bellman - Las ecuaciones óptimas de Bellman, caracterizan la función valor Vπ* de la política óptima π*:
 
 <br>
 
 **42. Remark: we note that the optimal policy π∗ for a given state s is such that:**
 
-&#10230;
+&#10230; Observación: denotamos que la política óptima π* para un estado dado s es tal que:
 
 <br>
 
 **43. Value iteration algorithm ― The value iteration algorithm is in two steps:**
 
-&#10230;
+&#10230; Algoritmo de iteración valor - El algoritmo de iteración valor es en dos pasos:
 
 <br>
 
 **44. 1) We initialize the value:**
 
-&#10230;
+&#10230; 1) Inicializamos el valor:
 
 <br>
 
 **45. 2) We iterate the value based on the values before:**
 
-&#10230;
+&#10230; Iteramos el valor con base en los valores de antes:
 
 <br>
 
 **46. Maximum likelihood estimate ― The maximum likelihood estimates for the state transition probabilities are as follows:**
 
-&#10230;
+&#10230; Estimado por máxima verosimilitud - El estimado por máxima verosimilitud para las probabilidades de transición de estado son como se muestra a continuación:
```

Comment:
"_Estimación por máxima verosimilitud_", as it has been translated in other sheets.